app/models/metric.rb

```diff
@@ -25,4 +25,8 @@ class Metric < ApplicationRecord
 
   validates :interval, inclusion: { in: intervals.keys }
   validates :name, inclusion: { in: names.keys }
+
+  scope :today_daily_metrics, lambda {
```

Comment:
WDYT of using `->` instead of `lambda`?

Reply:
at first i did it like that, but the analysis throw me a reek warning

Reply:
Alright! Did you find why is giving you that error?

Reply:
sorry I got confused, was not a warning .. the warning was about the way i was doing the query for the ownable.. there is a lambda because the need of a block so i couldn't do something like `scope today_daily_metrics, -> {`
`code..`
`}`

i had to convert it into a lamba (Rubocop)